.github/workflows/design-system-ci.yml

```diff
@@ -0,0 +1,58 @@
+name: Design System CI and QA
+
+on:
+    workflow_dispatch:
+    push:
+        branches:
+            - staging-next
+        paths:
+            - 'frontend/**'
+    pull_request:
+        branches:
+            - staging-next
```

Comment:
```suggestion
            - staging-next
            - staging
```

lets do both!

Reply:
The only thing is that I think we should target `staging` for both `push` and `pull_request` (see my earlier comment)